delivery_deliverea/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 FactorLibre - Jorge Martínez <jorge.martinez@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Delivery Deliverea",
+    "summary": "Delivery Carrier implementation for Deliverea using their API",
+    "version": "16.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "FactorLibre, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "base_delivery_carrier_label",
+        "delivery_state",
+        "queue_job",
```

Comment:
This is also not very convenient. A more abstract `delivery_queue_job` should be done that intercepts any API call (being SEUR, Deliverea, or whatever, and enqueue it). Anyway, I don't think this is a good idea, as you don't have immediate feedback if the sending is registered, and thus this can lead to missing packages not being informed to the carrier, provoking business problems.

Reply:
Please remove this dependency then. As said, you can make a generic `delivery_queue_job` module that intercepts the delivery methods (`_send_shipper` and company), and enqueue them instead of doing it directly. Check https://github.com/OCA/bank-payment/blob/8.0/account_payment_transfer_reconcile_batch/models/payment_order.py for a similar technique.

Reply:
Okaay thanks.
I removed the dependency.